usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -370,8 +368,72 @@ def fetch_business_categories_by_legal_entity_id(legal_entity_id):
     return []
 
 
+def fetch_all_cfda_details(award):
+    queryset = TransactionFABS.objects.filter(transaction__award_id=award["id"]).values(
+        "cfda_number", "federal_action_obligation", "non_federal_funding_amount", "total_funding_amount"
+    )
+    cfdas = {}
+    for item in queryset:
+        # sometimes the transactions data has the trailing 0 in the CFDA number truncated, this adds it back
+        num = item["cfda_number"]
+        if len(num) < 6:
+            num += "0" * (6 - len(num))
```

Comment:
My bad adding the last comment on the commit instead of the branch. I guess my only remaining concern is that `if` checking for length instead of null.

Reply:
yeah i'm adding that in